go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/oglok/nimbess-cni
+
+go 1.12
```

Comment:
should this be 1.11? and should it also reference the nimbess/nimbess-cni repo?

Reply:
Done